packages/keyda-demo/src/lib/Context/index.js

```diff
@@ -3,30 +3,62 @@ import React, { createContext, useContext, useReducer } from 'react';
 const initialKeyState = {
   keyTimeList: [],
   userId: '',
+  trainCount: 0,
+  inputRef: {},
 };
```

Comment:
`trainCount` is for counting submission, and `inputRef` is for storing `ref` object from `input` at `Input` component.